release-team/role-handbooks/docs/README.md

```diff
@@ -350,6 +350,10 @@ Create the PRs needed to roll the docs to the new version on the dev branch
 
 1. Deprecate the oldest API link in the reference docs, e.g: https://github.com/kubernetes/website/pull/13467
 
+1. Update the index for the API reference docs, e.g: https://github.com/kubernetes/website/pull/14139
+
```

Comment:
Do you think more details are warranted:
- Deprecate the oldest API link and add a new link for this release version, ....
    There should be five links to the latest five releases.
- Update the API reference version, in the API reference index page, to match the release version, ...

Reply:
That's fine!
Is there a link to this handbook from a page in /docs/contribute? Good for docs people to stay involved.

Reply:
> I think the changes are minor enough that an example would suffice. However, I will defer to @MAKOSCAFEE for the final say. In the meantime, I think we can merge this "as is" and Barney can include it if he chooses with any of his revisions post-release.

Sure this is fine. Will add more examples on my revisions.